.github/workflows/buildAndTest.yml

```diff
@@ -98,6 +104,7 @@ jobs:
         TM_PACKAGES="${{ matrix.llvm-build }}" \
         TM_USE_PYTORCH_BINARY="${{ matrix.torch-binary }}" \
         TM_PYTORCH_INSTALL_WITHOUT_REBUILD="${{ steps.cache-pytorch.outputs.cache-hit }}" \
+        TORCH_VERSION="${{ matrix.torch-version }}" \
```

Comment:
nit: we should probably keep the variables with TM_ and TM_TORCH_VERSION for the variables passed into the .sh script